src/test/crimson/test_messenger.cc

```diff
@@ -311,12 +311,13 @@ static seastar::future<> test_concurrent_dispatch(bool v2)
         switch (++count) {
         case 1:
           // block on the first request until we reenter with the second
-          return on_second.get_future()
+          on_second.get_future()
```

Comment:
`test_concurrent_dispatch()` was originally designed to verify that the second message can be dispatched even if the first one is blocked. So this test can be removed, right?

Reply:
@cyx1231st agreed. probably, instead of removing this test, we could repurpose it for "test_sequential_dispatch".  namely, 

1. increment `test_state::count` when entering `test_state::ms_dispatch()` and decrement it when leaving this method
1. add a `seastar::sleep(a_while)` in between increasing `count` and decreasing it`
1. check if the `count` is sane before changing it,

Reply:
Done, used `blocked` to check the sequential dispatching.